Shared/ViewModels/AdminDashboard/DevicesViewModel.swift

```diff
@@ -152,11 +94,9 @@ final class DevicesViewModel: ViewModel, Eventful, Stateful {
         let request = Paths.deleteDevice(id: id)
         try await userSession.client.send(request)
 
-        try await loadDevices()
+        try await _refresh()
```

Comment:
We should not be refreshing after we delete each individual item - we are refreshing after we do the entire list.